Pipfile

```
@@ -21,7 +21,7 @@ markdown = "==3.4.4"
 marshmallow = "==3.20.1"
 networkx = "==2.6"
 openpyxl = "==3.1.2"
-pandas = ">=1.3.5,<2.3.0"
+pandas = ">=1.3.5"
```

Let's always use this version of pandas. The Pipfile.max test will be tried above.
```suggestion
pandas = "==1.3.5"
```